src/services/ui/src/pages/welcome/index.tsx

```diff
@@ -39,37 +39,62 @@ export const Welcome = () => {
               <C.HowItWorks>
                 <C.Step
                   icon={<AcademicCapIcon className="min-w-[32px] w-8 h-8" />}
-                  title="Possimus a, odio."
-                  content="Lorem ipsum dolor sit amet."
+                  title="Login with IDM"
+                  content="Login with your IDM username and password to access your SPA and Waiver dashboard."
                 />
                 <C.Step
                   icon={<AcademicCapIcon className="min-w-[32px] w-8 h-8" />}
-                  title="Possimus a, odio."
-                  content="Lorem ipsum dolor sit amet."
+                  title="Attach your documents"
+                  content="Select a submission type and attach required documents relevant to your SPA and/or Waiver submission."
                 />
                 <C.Step
```

Comment:
can we update these with more appropriate icons based on what is used in onemac?